14-ambarom-vignette.Rmd

```diff
@@ -362,7 +362,7 @@ While the table effectively presents the data, a map could also be insightful. T
 ```{r}
 #| label: ambarom-americas-map
 #| fig.cap: "Map of North and South America"
-#| error: true
+#| error: true 
```

Comment:
why are we outputting the error messages for all the maps?  It is very odd to see in the PDF without any explanation.

Reply:
When I compile, I don't see any errors